config/_shared.json

```diff
@@ -1,3 +1,170 @@
 {
-	"discover_blog_id": 53424024
+	"discover_blog_id": 53424024,
+	"muse_supported_themes": [
+		"pub/twentytwelve",
+		"pub/twentythirteen",
+		"pub/twentyfourteen",
+		"pub/together",
+		"pub/capoverso",
+		"pub/edin",
+		"pub/musuem",
+		"pub/sequential",
+		"pub/minnow",
+		"pub/penscratch",
+		"pub/intergalactic",
+		"pub/goran",
+		"pub/sketch",
+		"pub/eighties",
+		"pub/editor",
+		"pub/kelly",
+		"pub/chateau",
+		"pub/espied",
+		"pub/big-brother",
+		"pub/ryu",
+		"pub/illustratr",
+		"pub/hexa",
+		"pub/harmonic"
+	],
+	"languages": [
+		{ "value": 2, "langSlug": "af", "name": "af - Afrikaans" },
+		{ "value": 418, "langSlug": "als", "name": "als - Alemannisch" },
+		{ "value": 481, "langSlug": "am", "name": "am - አማርኛ" },
+		{ "value": 3, "langSlug": "ar", "name": "ar - العربية", "rtl": true },
+		{ "value": 419, "langSlug": "arc", "name": "arc - ܕܥܒܪܸܝܛ" },
+		{ "value": 4, "langSlug": "as", "name": "as - অসমীয়া" },
+		{ "value": 420, "langSlug": "ast", "name": "ast - Asturianu" },
+		{ "value": 421, "langSlug": "av", "name": "av - Авар" },
+		{ "value": 422, "langSlug": "ay", "name": "ay - Aymar" },
+		{ "value": 79, "langSlug": "az", "name": "az - Azərbaycan" },
+		{ "value": 423, "langSlug": "ba", "name": "ba - Башҡорт" },
+		{ "value": 5, "langSlug": "be", "name": "be - Беларуская" },
+		{ "value": 6, "langSlug": "bg", "name": "bg - Български" },
+		{ "value": 7, "langSlug": "bm", "name": "bm - Bamanankan" },
+		{ "value": 8, "langSlug": "bn", "name": "bn - বাংলা" },
+		{ "value": 9, "langSlug": "bo", "name": "bo - བོད་ཡིག" },
+		{ "value": 424, "langSlug": "br", "name": "br - Brezhoneg" },
+		{ "value": 454, "langSlug": "bs", "name": "bs - Bosanski" },
+		{ "value": 10, "langSlug": "ca", "name": "ca - Català" },
+		{ "value": 425, "langSlug": "ce", "name": "ce - Нохчийн" },
+		{ "value": 11, "langSlug": "cs", "name": "cs - Česky" },
+		{ "value": 12, "langSlug": "csb", "name": "csb - Kaszëbsczi" },
+		{ "value": 426, "langSlug": "cv", "name": "cv - Чӑваш" },
+		{ "value": 13, "langSlug": "cy", "name": "cy - Cymraeg" },
+		{ "value": 14, "langSlug": "da", "name": "da - Dansk" },
+		{ "value": 15, "langSlug": "de", "name": "de - Deutsch", "popular": 4 },
+		{ "value": 427, "langSlug": "dv", "name": "dv - ދިވެހިބަސް" },
```

Comment:
Can someone double check if that works properly for users? Everything else displays properly for me, but this one seems to be broken.